indra/newview/llpuppetmotion.cpp

```diff
@@ -1241,6 +1239,10 @@ void LLPuppetMotion::onDeactivate()
     // this motion from its active list.
     mPose.removeAllJointStates();
     mJointsToRemoveFromPose.clear();
+    for (auto& data_pair : mJointStates)
+    {
+        data_pair.second->setUsage(0);
+    }
```

Comment:
Clear the usage on deactivation (and when removing the pose further up).  The reason we want to clear it when possible is because the animation blending checks the usage flags to figure out what math needs to be done: leaving extraneous usage flags around would make the blending calculations do unnecessary work.